tests/test_timers.cpp

```diff
@@ -154,21 +154,34 @@ int main (void)
 
     bool timer_invoked = false;
 
-    int timer_id = zmq_timers_add (timers, 100, handler, &timer_invoked);
+    const int full_timeout = 100;
+    void *const stopwatch = zmq_stopwatch_start ();
+
+    int timer_id =
+      zmq_timers_add (timers, full_timeout, handler, &timer_invoked);
     assert (timer_id);
 
-    //  Timer should be invoked yet
+    //  Timer should not have been invoked yet
     int rc = zmq_timers_execute (timers);
     assert (rc == 0);
-    assert (!timer_invoked);
+
+#ifdef ZMQ_BUILD_DRAFT_API
+    if (zmq_stopwatch_intermediate (stopwatch) < full_timeout) {
```

Comment:
@sigiesec with older versions of gcc there's this error:

```
[  220s] cc1plus: warnings being treated as errors
[  220s] tests/test_timers.cpp: In function 'int main()':
[  220s] tests/test_timers.cpp:169: error: comparison between signed and unsigned integer expressions
[  220s] tests/test_timers.cpp:181: error: comparison between signed and unsigned integer expressions
```

Do you know what the issue is? Otherwise I'll try to find time over the weekend